test/utils.test.js

```diff
@@ -921,3 +921,26 @@ describe('utils.parseContentTypeFromResponse', () => {
         expect(parsedReallyBad.charset).toBe('utf-8');
     });
 });
+
+describe('utils.purgeLocalStorage()', () => {
+    // Create test folder
+    const folder = Date.now().toString(16);
+    fs.mkdirSync(folder);
+
+    test('should purge local storage by default', () => {
+        expect(() => utils.purgeLocalStorage()).not.toThrowError();
+
+        expect(fs.existsSync('apify_storage')).toBe(false);
+    });
+
+    test('should purge local storage when passing custom name', async () => {
+        try {
```

Comment:
do we need the try catch? if it would fail, we would end up with a failed test anyway, or not?

Reply:
there's a high probability that the function never throws (since thats how rimraf works), so realistically the try/catch is redundant

Reply:
My point is that even when it throws, it would fail the test on its own.